.travis.yml

```diff
@@ -3,13 +3,18 @@ sudo: required
 services:
   - docker
 
+env:
+  - SOLR_HOST=my_solr:8983
+
 before_install:
-  - docker run --name my_solr -d -p 8983:8983 -t solr:7.3-alpine solr-create -c scxa-analytics-v2
+  - docker network create mynet
+  - docker run --net mynet --name my_solr -d -p 8983:8983 -t solr:7.3-alpine solr-create -c scxa-analytics-v2
```

Comment:
Shouldn’t the `solr-create -c scxa-analytics-v2` be moved to the `script` section?

Reply:
Yeah, we could, is probably more correct in terms of travis usage. Thanks!